samples/core/parameterized_tfx_oss/parameterized_tfx_oss.py

```diff
@@ -24,16 +24,16 @@
 # or a module file baked in the docker image used by the pipeline.
 _taxi_module_file_param = tfx.dsl.experimental.RuntimeParameter(
     name='module-file',
-    default='/tfx/src/tfx/examples/chicago_taxi_pipeline/taxi_utils.py',
+    default='/opt/conda/lib/python3.7/site-packages/tfx/examples/chicago_taxi_pipeline/taxi_utils_native_keras.py',
     ptype=str,
 )
 
 # Path to the CSV data file, under which their should be a data.csv file.
-_data_root = 'gs://ml-pipeline-playground/tfx_taxi_simple/data'
+_data_root = '/opt/conda/lib/python3.7/site-packages/tfx/examples/chicago_taxi_pipeline/data/simple'
 
 # Path of pipeline root, should be a GCS path.
 pipeline_root = os.path.join(
-    'gs://{{kfp-default-bucket}}', 'tfx_taxi_simple', kfp.dsl.RUN_ID_PLACEHOLDER
```

Comment:
This is auto filled in marketplace installation, but not for standalone

Reply:
Oh! That was the difference. I'll rollback to '{{kfp-default-bucket}}'. Thanks!